ob-pattern-1/templates/wso2-ob-pattern-1-obam-volume-claims.yaml

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
@kdanW please update the license header year.

Reply:
Fixed in https://github.com/wso2/kubernetes-open-banking/pull/22/commits/d3e36e4952af558eb192f5ce2391d848f11cabb0